lib/bash/rose_init

```diff
@@ -31,7 +31,7 @@
 #     * load any FUNC specified in the argument list.
 #-------------------------------------------------------------------------------
 function rose_readlink_canonicalize() { # "readlink -f" is GNU extension
-    readlink -f $1 2>/dev/null || python2 - $1 <<'__PYTHON__'
+    readlink -f $1 2>/dev/null || python3 - $1 <<'__PYTHON__'
 import os, sys
 print os.path.realpath(sys.argv[1])
```

Comment:
The 'heredoc' Python lines themselves have been missed for conversion in this case:
```suggestion
print(os.path.realpath(sys.argv[1]))
```

Reply:
Thank you! I've committed your change!